mindsdb/integrations/handlers/gcs_handler/README.md

```diff
@@ -19,17 +19,20 @@ CREATE DATABASE gcs_datasource
 WITH
```

Comment:
Can you please update the documentation based on  the changes made here?
https://github.com/mindsdb/mindsdb/pull/9979

Reply:
There's no markdown files for gcs in the docs folder. Shall I create one?

Reply:
We can tackle that later. For now, please post screenshots of this feature working.

Reply:
![6122765859916200658](https://github.com/user-attachments/assets/1208d395-449e-4186-84c5-20c2dd116781)

![6122765859916200659](https://github.com/user-attachments/assets/450286a1-e448-475d-86f5-49dcbdf08e4e)

![6122765859916200662](https://github.com/user-attachments/assets/2c10df0a-2c46-4ffb-858d-97eab07694a0)

It's working fine.

Reply:
Please also post a screenshot of querying a file in a bucket providing the full path.
E.g., `my-bucket/my-dir/my-file.csv`

Reply:
![6122765859916200864](https://github.com/user-attachments/assets/f854fd66-4cae-47e5-b5a6-623ef73fb6dc)

Done.  Tested.